config/locales/controller/fitlogs.ja.yml

```
@@ -0,0 +1,7 @@
+ja:
+  create:
+      notice: '記録が保存出来ました'
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)